packages/@react-spectrum/s2/src/Skeleton.tsx

```diff
@@ -120,7 +120,7 @@ export function SkeletonWrapper({children}: {children: SkeletonElement}): ReactN
     return children;
   }
 
-  let childRef = 'ref' in children ? children.ref as any : children.props.ref;
+  let childRef = 'ref' in children && !Object.getOwnPropertyDescriptor(children, 'ref')?.get ? children.ref as any : children.props.ref;
```

Comment:
Fixes crash in storybook in React 19. React emits a warning when accessing `element.ref`. This checks if that is defined as a getter.